scripts/monitoring/ops-metric-client.py

```diff
@@ -64,6 +65,21 @@ def run(self):
 
         self.metric_sender.send_metrics()
 
+    @staticmethod
+    def adjust_type(val, metric_type):
+        """
+        :param val: string representing metric value
+        :param metric_type: string or numeric
+        :return: value converted to float if metric_type isn't string
+        """
+        if metric_type == 'string':
+            return val
+
+        try:
+            return json.loads(val)
```

Comment:
@twiest I can try setting up zabbix on Sunday, but it will probably take me a long time, and most likely I will need assistance from someone from your team.
It will probably go much faster if someone on your side could just pull the PR and test locally.

P.S.
running with `-m string` imitates old script behaviour